### PR TITLE
Added entity type when viewing trash can 

### PR DESCRIPTION
**Pull request by linglp**
Related to https://sagebionetworks.jira.com/browse/SWC-6367 
I added entity type on trash can page. 

More context: 
I tried making a synapse api call using synapse python client to endpoint: https://rest-docs.synapse.org/rest/GET/trashcan/view.html
By using my own synapse PAT, I could see that the response object is something like: 
```
[{'entityId': 'syn52920298', 'entityName': 'mockrdb_synapse_storage_manifest_table', 'entityType': 'table', 'deletedByPrincipalId': '3443707', 'deletedOn': '2023-11-09T17:58:42.000Z', 'originalParentId': 'syn52854552'}, {'entityId': 'syn52854759', 'entityName': 'mockrdb_synapse_storage_manifest_table', 'entityType': 'table', 'deletedByPrincipalId': '3443707', 'deletedOn': '2023-11-09T17:56:49.000Z', 'originalParentId': 'syn52854552'}]
```
This seems to suggest that entityType is string and it doesn't require further processing.

### CODE REVIEW

File: packages/synapse-react-client/src/components/trash/TrashCanList.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ function TrashCanListItem(props: TrashCanListItemProps) {
       </td>
       <td>{item.entityId}</td>
       <td>{item.entityName}</td>
+      <td>{item.entityType}</td>
```
Comment (nickgros) [inline packages/synapse-react-client/src/components/trash/TrashCanList.tsx:41]: Use `entityTypeToFriendlyName` from EntityTypeUtils.ts

File: packages/synapse-types/src/TrashedEntity.ts [hunk 2]
```diff
@@ -3,11 +3,15 @@
  *
  * https://docs.synapse.org/rest/org/sagebionetworks/repo/model/TrashedEntity.html
  */
+import { EntityType } from './EntityType'
+
```
Comment (nickgros) [inline packages/synapse-types/src/TrashedEntity.ts:7]: Please move this import to the top of the file, so the above comment is applied to the `TrashedEntity` type

**Review by nickgros — CHANGES_REQUESTED** [review]

File: packages/synapse-react-client/src/components/trash/TrashCanList.tsx (hunk 1 above)
Comment (linglp) [reply packages/synapse-react-client/src/components/trash/TrashCanList.tsx]: Thanks Nick! i just updated that.

File: packages/synapse-types/src/TrashedEntity.ts (hunk 2 above)
Comment (linglp) [reply packages/synapse-types/src/TrashedEntity.ts]: Thanks @nickgros ! I just moved that line to the top

**Review by nickgros — APPROVED** [review]

**Review by nickgros — CHANGES_REQUESTED** [review]
@linglp this looks great, but it looks like there are type errors in TrashCanList.integration.test.tsx . Can you investigate

**Comment by linglp** [comment]
> @linglp this looks great, but it looks like there are type errors in TrashCanList.integration.test.tsx . Can you investigate?

Thanks Nick. Is there a way to run all the integration tests locally? I modified the test by adding entityType. I want to double check if the tests are passing now.

**Review by nickgros — APPROVED** [review]
Thanks @linglp ! Nice work!

**Pull request merged**